CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
-## 7.0.1 (Next)
+## 7.1.1 (Next)
 
-* Your contribution here.
+## 7.1.0 (2023/10/20)
```

Comment:
We haven't released 7.1.0. Let's increment the version to 7.1.0 in this PR, it should say `7.1.0 (Next)`.